### PR TITLE
💍 Rename ChannelSession to ChannelConnection

### PR DESCRIPTION
**Pull request by rowanseymour**
Addresses https://github.com/rapidpro/rapidpro/issues/839

### CODE REVIEW

File: temba/channels/migrations/0111_rename_channelsession.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Generated by Django 2.1.3 on 2019-01-10 20:29
+
+from django.db import migrations
+
+
+class RenameBaseModel(migrations.RenameModel):
```
Comment (rowanseymour) [inline temba/channels/migrations/0111_rename_channelsession.py]: Currently RenameModel doesn't work with inheritance.. below is the code from the regular RenameModel with extra code under the `# repoint bases..` comment. Not ideal having this but I'd assume we  will merge migrations before upgrading to a new Django

**Comment by nicpottier** [comment]
I know we talked about `Connection` vs `ChannelConnection` but now that I'm seeing it I'm reconsidering just from a consistency perspective. IE, we have `FlowSession` instead of `Session` and `FlowRun` instead of `Run`, should `Connection` be any different?

**Comment by rowanseymour** [comment]
I'm fine with that as long as the related field names are allowed to be simpler - i.e. `run.session` `msg.session`, `msg.connection`, `contact.connections`

**Comment by nicpottier** [comment]
Don't feel strongly about related names (and more easily changeable) so sure.

**Comment by rowanseymour** [comment]
@nicpottier renamed - should be able to push this later this afternoon if it all looks ok

**Comment by nicpottier** [comment]
Nice, thanks for doing that (and apologies for the waffling)

**Comment by nicpottier** [comment]
@rowanseymour  while you are in here want to add the M2M and make `created_by` and `modified_by` nullable?

**Comment by nicpottier** [comment]
I'll just open a new PR off of this branch actually.

File: temba/channels/models.py [hunk 2]
```diff
@@ -2193,7 +2193,11 @@ def get_alert_user():
         return user
 
 
-class ChannelSession(models.Model):
+class ChannelConnection(models.Model):
```
Comment (nicpottier) [inline temba/channels/models.py]: I think we also need to rename `session_type` to `connection_type`.

**Comment by nicpottier** [comment]
Lets do this!

**Pull request merged**